websockets/protocol.py

```diff
@@ -313,15 +315,15 @@ def recv(self):
             next_message.cancel()
             raise
 
-        # Now there's no need to yield from self.ensure_open(). Either a
-        # message was received or the connection was closed.
-
         if next_message in done:
             return next_message.result()
         else:
             next_message.cancel()
             if not self.legacy_recv:
-                raise ConnectionClosed(self.close_code, self.close_reason)
+                assert self.state in [State.CLOSING, State.CLOSED]
```

Comment:
I wonder whether use a tuple instead of a list here is considered better practice?

```python
assert self.state in (State.CLOSING, State.CLOSED)
```

Reply:
Well, that does make sense.